lib/bots/versionbot.ts

```diff
@@ -900,16 +906,16 @@ export class VersionBot extends GithubBot.GithubBot {
         });
     }
 
-    private validMaintainer(config: VersionBotConfiguration, event: GithubBotApiTypes.PullRequestEvent): void {
+    private isValidMaintainer(config: VersionBotConfiguration, event: GithubBotApiTypes.PullRequestEvent): void {
```

Comment:
And again to be pedantic, I'd aslo expect a `is...` method to return a boolean, so perhaps `checkValidMaintianer` would be a better name here?